<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Timetable Planner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="scheduleTableView">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectItems</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionImportCSV"/>
    <addaction name="separator"/>
    <addaction name="actionExportCSV"/>
    <addaction name="actionExportText"/>
    <addaction name="actionExportPDF"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSchedule">
    <property name="title">
     <string>Schedule</string>
    </property>
    <addaction name="actionGenerate"/>
    <addaction name="actionViewSchedule"/>
    <addaction name="separator"/>
    <addaction name="actionAnalytics"/>
    <addaction name="separator"/>
    <addaction name="actionShiftCourse"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSchedule"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionImportCSV"/>
   <addaction name="actionGenerate"/>
   <addaction name="actionViewSchedule"/>
   <addaction name="actionAnalytics"/>
   <addaction name="separator"/>
   <addaction name="actionExportCSV"/>
   <addaction name="actionExportText"/>
   <addaction name="actionExportPDF"/>
  </widget>
  <action name="actionConnect">
   <property name="text">
    <string>Connect to Database</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionImportCSV">
   <property name="text">
    <string>Import from CSV</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
   <property name="toolTip">
    <string>Import student enrollment data from CSV file</string>
   </property>
  </action>
  <action name="actionGenerate">
   <property name="text">
    <string>Generate Timetable</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionViewSchedule">
   <property name="text">
    <string>View Schedule</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionExportCSV">
   <property name="text">
    <string>Export to CSV</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExportText">
   <property name="text">
    <string>Export to Text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionExportPDF">
   <property name="text">
    <string>Export to PDF</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionShiftCourse">
   <property name="text">
    <string>Shift Course</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAnalytics">
   <property name="text">
    <string>ðŸ“Š Analytics Dashboard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
   <property name="toolTip">
    <string>Open Analytics Dashboard with Charts and Statistics</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>